.github/workflows/docker-build-context/entrypoint-finalize-setup.bash

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Entrypoint script used by our custom EE server image
+
+# Bash required for using job control
+
+set -x
+set -m
+set -e
+
+asd --fgdaemon --config-file $AEROSPIKE_CONF_PATH &
```

Comment:
I know shellcheck or shfmt wold complain here - in this case, probably unnecessarily... but it these tools have protected my scripts from surprising bash behaviors (at least to me) in the past.

https://github.com/aerospike/aerospike-server.docker/blob/version-dirs/.github/workflows/reviewdog.yml

Reply:
Thx -- I'll run it next time. I didn't see anything break when testing the scripts, so I think it should be ok for now